app/controllers/application_controller.rb

```diff
@@ -19,13 +19,14 @@ def prepare_for_mobile
 
   def setup_negative_captcha
     @captcha = NegativeCaptcha.new(
-      # A secret key entered in environment.rb. 'rake secret' will give you a good one.
+      # A secret key entered in environment.rb. 'rake secret' will give
+      # you a good one.
       secret: ENV['NEGATIVE_CAPTCHA_SECRET'],
       spinner: request.remote_ip,
       # Whatever fields are in your form
       fields: [:name, :content, :photo],
       params: params,
-      message: "X__x"
+      message: 'X__x'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.